packages/single-selection-mixin/single-selection-class.ts

```diff
@@ -0,0 +1,7 @@
+import { LitElement } from 'lit-element';
+
+export abstract class SingleSelectionClass extends LitElement {
+  protected _onClick?(event: MouseEvent): void;
+
+  protected _onKeyUp?(event: KeyboardEvent): void;
```

Comment:
Just started thinking that why do we have this kind of event handlers (with very generic name) as protected API?

I think this can cause problems if the user/dev provides a base class that already has an event handler with similar generic name and this can also soon become a problem/conflict in our own classes.

Currently these implementations don't check for possible super method, but even if they would, I think it's somehow weird that we have these generic event name based methods as part of an abstract class that should define the API for this mixin behaviour, but these method names don't seem to be any way related to the mixin behaviour in my opinion.

Should we handle these in some other way?

Reply:
Submitted #25